setup.cfg

```diff
@@ -72,3 +72,4 @@ exclude_lines =
   def _ipython_key_completions_
 
 [entry_points]
+samp_hub = pyvo.astropy_samp.hub_script:hub_script
```

Comment:
I am not sure how confusing this CLI will get since `astropy` and `pyvo` will have this same command until we manage to remove the code from `astropy` in a future release (it needs a deprecation period over there).